lib/exp.js

```diff
@@ -29,29 +29,76 @@ const BIN_TYPE_UNDEF = 0
  * VALUE EXPRESSIONS
  *********************************************************************************/
 
-const _valueExp = (op, valName) => (val) => [{ op, [valName]: val }]
+const _valueExp = (op, valName) => (value) => [{ op, [valName]: value }]
 
-const _valueExpBytes = (op, valName, sizeName) => (val, size) => [{ op, [valName]: val, [sizeName]: size }]
+const _valueExpBytes = (op, valName, sizeName) => (value, size) => [{ op, [valName]: value, [sizeName]: size }]
 
+/**
+ * Create boolean value.
+ *
+ * @param value boolean value.
```

Comment:
Would be best to specify the parameter type as well, using e.g.

```
@param {boolean} value - boolean value.
```

Ref. https://jsdoc.app/tags-param.html

Reply:
Is this still an issue? Looks like the types need curly braces?